tests/integ/modin/index/test_name.py

```diff
@@ -317,3 +317,39 @@ def test_index_SNOW_1021837():
     native_df_reset = native_df.reset_index()
     snow_df_reset = snow_df.reset_index()
     assert_frame_equal(snow_df_reset, native_df_reset)
+
+
+@sql_count_checker(query_count=0)
+def test_index_non_list_like_names():
+    """
+    Bug SNOW-1650853:
```

Comment:
Not sure there's much value in documenting historical behavior of a bug like this here in the test.

Reply:
removed it!